hw_diag/templates/diagnostics_page.html

```diff
@@ -92,38 +66,46 @@ <h3 class="text-center mb-4">Diagnostics Breakdown</h3>
                 </tr>
                 <tr>
                   <td>Region Plan</td>
-                  {% if diagnostics.RE == 'UN123' %}
-                    <td class="text-right">Awaiting Location Assertion</td>
+                  {% if diagnostics.miner_region %}
```

Comment:
as mentioned [here](https://github.com/NebraLtd/hm-diag/pull/337#discussion_r858668223) I can't do check errors because only plain dict is exposed to the template. I am trying creating DiagnosticReport out of json and expose that. Will update PR if that works.

Reply:
now DiagnosticReport is exposed instead of plain dict. There doesn't seem to be any regression because of that and it will correctly show disconnected.